src/views/Settings.vue

```diff
@@ -125,6 +125,15 @@
               :prepend-icon="mdiWeatherNight"
             />
             <!-- :messages="$t('views.settings.darkModeMsg')" -->
+            <v-switch
+              v-model="watchViewReversed"
+              class="v-input--reverse v-input--expand"
+              :label="$t('views.settings.watchViewReversedLabel')"
+              hide-details
+              inset
+              :prepend-icon="mdiWeatherNight"
```

Comment:
I'm not sure what icon is appropriate here.

Reply:
Perhaps `mdiFlipHorizontal`? You can search through the material design icons on sites like https://mdisearch.com/.

Reply:
also suggest changing the watch button to represent the direction the chat dock will happen in `mdiDockLeft` vs `mdiDockRight`